assets/quick-add.js

```diff
@@ -35,6 +35,7 @@ if (!customElements.get('quick-add-modal')) {
           if (window.ProductModel) window.ProductModel.loadShopifyXR();
 
           this.removeGalleryListSemantic();
+          this.updateImgSizes();
```

Comment:
small nitpick, I remember being told once not to shorten words for clarity and readability, other functions I think are the same so here we could go with `updateImageSizes()`